src/libs/OptionsListUtils.ts

```diff
@@ -1005,10 +1026,12 @@ function orderOptions(options: ReportAndPersonalDetailOptions, searchValue?: str
         orderedReportOptions = orderReportOptions(options.recentReports);
     }
     const orderedPersonalDetailsOptions = orderPersonalDetailsOptions(options.personalDetails);
+    const orderedRecentWorkspaceChats = orderWorkspaceOptions(options?.recentWorkspaceChats ?? []);
```

Comment:
The recent workspaces maybe don't include primary workspace (activePolicyID) in cases where we have many workspaces.